onnxruntime/core/providers/nnapi/nnapi_builtin/builders/helper.h

```diff
@@ -132,5 +132,9 @@ std::vector<std::vector<size_t>> GetSupportedNodes(const GraphViewer& graph_view
 // Get string representation of a Shape
 std::string Shape2String(const std::vector<uint32_t>& shape);
 
+// Check the given input is an initializer tensor
+bool CheckIsInitializerTensor(const InitializedTensorSet& initializers, const Node& node,
+                              size_t index, const char* input_name) ORT_MUST_USE_RESULT;
```

Comment:
nit: all initializers are tensors so could be CheckIsInitializer

Reply:
Will change in my next PR